Userland/Applications/Presenter/SlideObject.cpp

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2022, kleines Filmröllchen <filmroellchen@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "SlideObject.h"
+#include "AK/kmalloc.h"
+#include "LibCore/Stream.h"
+#include "LibGUI/Margins.h"
+#include "LibGfx/Forward.h"
+#include "LibGfx/Orientation.h"
+#include "LibImageDecoderClient/Client.h"
+#include <AK/JsonObject.h>
+#include <AK/RefPtr.h>
+#include <LibCore/Object.h>
+#include <LibGfx/Font/FontDatabase.h>
+#include <LibGfx/Painter.h>
+#include <LibGfx/Size.h>
+#include <LibGfx/TextWrapping.h>
+#include <utility>
```

Comment:
This include is from c++'s std and makes the clang build fail

Reply:
clangd strikes again!